ql/src/test/results/clientpositive/llap/order_by_expr_2.q.out

```diff
@@ -139,7 +139,7 @@ POSTHOOK: Input: default@store
 CBO PLAN:
 HiveProject(c0=[$0], c1=[$1], c2=[$2])
   HiveSortLimit(sort0=[$3], dir0=[ASC])
-    HiveProject(c0=[$0], c1=[$1], c2=[$2], (tok_function case (tok_table_or_col store_name) 'HQ' tok_null (tok_table_or_col store_name))=[$1])
+    HiveProject(c0=[$0], c1=[$1], c2=[$2], (tok_function when (= (tok_table_or_col store_name) 'HQ') tok_null (tok_table_or_col store_name))=[$1])
```

Comment:
It took me a few minutes to understand that this is an alias that is identical to AST :)